tools/CMakeLists.txt

```diff
@@ -1,12 +1,11 @@
 set(SUBSYS_NAME tools)
 set(SUBSYS_DESC "Useful PCL-based command line tools")
-set(SUBSYS_DEPS common io filters sample_consensus segmentation search kdtree features surface octree registration recognition geometry keypoints ml)
-set(SUBSYS_OPT_DEPS vtk visualization)
+set(SUBSYS_OPT_DEPS common io filters sample_consensus segmentation search kdtree features surface octree registration recognition geometry keypoints ml visualization vtk)
```

Comment:
On second thought, how about we keep `io` as a "hard" dependency since all tools (well, except one) need it?